frame/atomic-swap/src/lib.rs

```diff
@@ -15,7 +15,27 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-//! # Atomic swap support pallet
+//! # Atomic Swap
+//!
+//! A module for atomically sending funds.
+//!
+//! - [`scheduler::Trait`](./trait.Trait.html)
```

Comment:
```suggestion
//! - [`atomic-swap::Trait`](./trait.Trait.html)
```

Reply:
Might be underscore, i dunno

Reply:
Gah, thank you...believe it is an underscore.